src/app.js

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2015 Colin Clark
+
+Licensed under the 3-Clause "New" BSD license.
+You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the 3-Clause "New" BSD License at
+https://github.com/colinbdclark/infusion-electron/raw/master/LICENSE.txt
+*/
+
+"use strict";
+
+var fluid = require("infusion"),
+    electron = fluid.registerNamespace("electron");
+
+// Monkey-patch fluid.log for Electron's broken console.log.
+fluid.doLog = function (args) {
+    if (typeof (console) !== "undefined") {
+        if (console.debug) {
+            console.debug(args.join(" "));
+        } else if (typeof (console.log) === "function") {
+            console.log(args.join(" "));
```

Comment:
If it's broken by virtue of not being able to log objects, you may like to apply fluid.prettyPrintJSON to args of type "object" before concatenating here